### PR TITLE
Update tests README

### PR DESCRIPTION
**Pull request by IljaDaderko**
Updates `tests/README.md` to reflect new scripts and adds some additional info

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -18,18 +17,26 @@ Our tests are powered by [Jet ✈️](https://github.com/invertase/jet).
...
     ```
 
+> **Note**: If Homebrew complains about a conflict in the `wix/brew` tap, run `brew untap wix/brew && brew tap wix/brew` and try installing again
+
 ---
 
-### Step 1: Install test project dependencies
+## Cleaning dependencies
+
+You might find yourself in situation where you want to start from a clean slate. Following will delete node_modules and project build folders.
```
Comment (Salakar) [inline tests/README.md]:
```suggestion
You might find yourself in a situation where you want to start from a clean slate. The following will delete all `node_modules` and project `build` folders.
```

File: tests/README.md [hunk 2]
```diff
@@ -59,18 +66,27 @@ As always; the first build for each platform will take a while. Subsequent build
...
 
 ---
 
-### Step 4: Finally, run the tests
+### Step 4: Setting up android emulator and iOS simulator
+
+To run android tests you will need to create a new emulator and name it `TestingAVD` (You can't rename existing one).
+This emulator will need to be up and running before you start your android tests from Step 5.
+
+With iOS Detox will start simulator for you by default or run tests in open one.
```
Comment (Salakar) [inline tests/README.md]:
```suggestion
With iOS Detox will start a simulator for you by default or run tests in an open one.
```

**Comment by Salakar** [comment]
With suggested changes this LGTM 👍

**Comment by mikehardy** [comment]
Sweet!

**Pull request merged**